cypress/integration/iframe.spec.ts

```diff
@@ -0,0 +1,21 @@
+import {IframePage} from "../page/index";
+
+
+describe("the user navigates to the Iframe  page should", () => {
+  let iframePage: IframePage;
+
+  before(() => {
+    iframePage = new IframePage();
+  });
+
+  it("show form and fill the personal form ", () => {
+    // arrange
+    iframePage.visit();
+    // action
+    iframePage.getTitle();
+    iframePage.goToCssPageInFrame();
+    // asserts
+    iframePage.getTitle().should("include.text", "HTML Tutorial");
+    iframePage.cssPageConfirmed().should("include.text", "CSS Tutorial");
+  });
```

Comment:
these should be two different tests (`it`).

First test to verify the HTML tutorial frame title, the second one to verify that you can open the CSS tutorial in the original frame.